src/string.cr

```diff
@@ -2402,6 +2402,25 @@ class String
     true
   end
 
+  # Returns `self` unless `#blank?` is `true` in which case it returns `nil`.
+  #
+  # ```
+  # "a".presence || "default"         # => "a"
+  # "".presence || "default"          # => "default"
+  # "   ".presence || "default"       # => nil
+  # "    a    ".presence || "default" # => "a"
+  # nil.presence || "default"         # => "default"
```

Comment:
I don't think these examples should use `|| "default"`. It's just clutter and not necessary to understand the behaviour of the method.

Having one example for this application is fine, but in general the sample code should be fully focused on the described feature.

Reply:
We can use other word than `"default"`, but I think this will become a used idiom. 

Having these cases here directly will be practical for the reader. In particular, I want them to not need to go to de definition of `blank?` to check and _deduce_ how many spaces will be handled.

Reply:
Yes, I want to keep the examples! Just strip off ` || "default"` from every line as it's not essential. These samples should primarily document what `#presence` returns, which is either self or nil.

Reply:
I agree with @straight-shoota here. Updated examples.